CMakeLists.txt

```diff
@@ -42,13 +42,15 @@ set(GFXRECONSTRUCT_PROJECT_VERSION_MAJOR 0)
 set(GFXRECONSTRUCT_PROJECT_VERSION_MINOR 9)
 set(GFXRECONSTRUCT_PROJECT_VERSION_PATCH 8)
 
-set(GFXRECON_PROJECT_VERSION_DESIGNATION "-unknown")
 set(GFXRECON_PROJECT_VERSION_SHA1 "unknown-build-source")
 
 if (NOT DEFINED HEADLESS)
     set(HEADLESS TRUE)
 endif()
 
+option(BUILD_ZSTD_SUPPORT "Build zstd support" OFF)
+option(BUILD_STATIC "Build static binaries (will not build tracer library)" OFF)
```

Comment:
I don't remember seeing this in the original PR.  What is the problem being solved here, and can this be a separate PR?

Reply:
It is for building binaries for HPC clusters. I've moved it into a separate PR.